### ISSUE TITLE
Performance of CIDEr decreases when performing self critic training at first 6000 iterations

### ISSUE THREAD

**Issue by vanpersie32** [ruotianluo/ImageCaptioning.pytorch#44]
Hi, I follow the instruction to train the model. Take the fc model for example. I train it with cross entropy loss for 25 epochs(iterations: 336000),  and the CIDEr on validation set is  0.92. Then I further train the model with scst, its CIDEr on validation set is 0.89 at iterations 342000. I want to ask why the cider  decrease  
at the first 6000 iterations of scst(self critic sequence training).
